src/brogue/Items.c

```diff
@@ -637,21 +642,33 @@ void populateItems(short upstairsX, short upstairsY) {
             // Guarantee a certain nutrition minimum of the approximate equivalent of one ration every four levels,
             // with more food on deeper levels since they generally take more turns to complete.
             theCategory = FOOD;
-            if (rogue.depthLevel > AMULET_LEVEL) {
+            if (rogue.depthLevel > gameConst.amuletLevel) {
                 numberOfItems++; // Food isn't at the expense of lumenstones.
             }
-        } else if (rogue.depthLevel > AMULET_LEVEL) {
+        } else if (rogue.depthLevel > gameConst.amuletLevel) {
             theCategory = GEM;
         } else {
-            // Guarantee any metered items that reach generation thresholds.
-            for (j = 0; j < NUMBER_METERED_ITEMS; j++) {
+            
+            //To preserve the ability to load RB 1.4 games, we need to create items in the same order as that codebase
+            const int j_iteration_order[] = { 14, 18, 0, 15, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 16, 17, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29};
```

Comment:
This is a hack to retain binary compatibility with RB 1.4 games. It is useful for regression testing with v1.4 RB games. However, for release, this can be removed and the RB version can be updated to v1.5, because dungeons will be different.